src/games/strategy/triplea/Constants.java

```diff
@@ -144,6 +144,7 @@
   String UNIT_TYPE_CARRIER = "carrier";
   String UNIT_TYPE_FIGHTER = "fighter";
   String UNIT_TYPE_BOMBER = "bomber";
+  String UNIT_TYPE_TAC_BOMBER = "tactical_bomber";
```

Comment:
If I understand what you're saying here, it's that you don't like global constants.

Can't say I'd agree with that. I'd even call it a weakness in Java that globals aren't properly catered for and you need to public statics.

But anyway, I can hard code it if that will make you happier.